templates/toil-cluster-role-and-policy.yaml

```diff
@@ -70,6 +71,23 @@ Resources:
             Resource:
               - !Sub "arn:aws:sdb:${AWS::Region}:${AWS::AccountId}:domain/${ClusterName}*"
               - !Sub "arn:aws:sdb:${AWS::Region}:${AWS::AccountId}:domain/toil-registry"
+  CapacityReservationPolicy:
```

Comment:
Was wondering if there is already a pre defined AWS managed policy for this?   Also this seems pretty generic policy so if there isn't an AWS builtin policy can we move it the common essentials.yaml template?  https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/essentials.yaml

Reply:
I looked for one and couldn't find it. I am happy to move this to some-other-infra though.

Reply:
actually, lets move it to https://github.com/Sage-Bionetworks/sandbox-infra/blob/master/templates/accounts.yaml  create a `CapacityReservationRole` and assign yourself to it.